tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -42,4 +49,49 @@ Dashboard Test Setup
   Wait for RHODS Dashboard to Load
 
 Dashboard Test Teardown
-  Close All Browsers
\ No newline at end of file
+  Close All Browsers
+  
+Get HTTP Status Code
```

Comment:
there is `Check HTTP Status Code` keyword in tests/Resources/page/ODH/ODHDashboard/ODHDashboard.robot. I think it is better to use it to avoid duplication